template/component.h.erb

```diff
@@ -136,17 +136,16 @@ struct common_data_s {
 <% end %>
 
 /* Implemented by params.c */
-void btx_read_params(common_data_t *common_data, params_t *usr_params);
+void btx_read_params(void *btx_handle, params_t *usr_params);
```

Comment:
Why sometimes `btx_handle_t` and others `void`?

Reply:
Always `void *`, `btx_handle_t` is a mistake that I made in the example and display. Has to tired to fix, but will do

Reply:
I can do it, to familiarize with the new naming.